src/containers/audio-rewards-page/AudioRewardsPage.tsx

```diff
@@ -6,15 +6,38 @@ import Page from 'components/general/Page'
 import styles from './AudioRewardsPage.module.css'
 import WalletModal from './WalletModal'
 import Tiers from './Tiers'
-import { ClaimTile, WalletTile } from './Tiles'
+import { BalanceTile, WalletTile } from './Tiles'
 import ExplainerTile from './components/ExplainerTile'
+import RewardsTile from './RewardsTile'
+import { isMobile } from 'utils/clientUtil'
+import { useMobileHeader } from 'components/general/header/mobile/hooks'
+import MobilePageContainer from 'components/general/MobilePageContainer'
+import { AUDIO_PAGE, BASE_URL } from 'utils/route'
+import { useWithMobileStyle } from 'hooks/useWithMobileStyle'
 
 export const messages = {
   title: '$AUDIO & Rewards',
   description: 'View important stats like plays, reposts, and more.'
 }
 
-export const AudioRewardsPage = () => {
+export const RewardsContent = () => {
+  const wm = useWithMobileStyle(styles.mobile)
+
+  return (
+    <>
+      <ExplainerTile className={wm(styles.explainerTile)} />
+      <WalletModal />
+      <div className={wm(styles.cryptoContentContainer)}>
+        <BalanceTile className={wm(styles.balanceTile)} />
```

Comment:
I see how the func `useWithMobileStyle` works, so the `.mobile` classname is used with every `wm`  instance if it is mobile. 
What are your thoughts on using just a conditional `.mobile` only at the top level and each className instead of having .mobile in it, just uses the .mobile selector from the parent in the css file.

Reply:
Discussed on slack - I like the top level .mobile, will try to do that going forward when possible